automation/patch_igz/patch_remote.py

```diff
@@ -386,90 +365,72 @@ def _wait_deployment_ready(self):
                 "app.kubernetes.io/name=mlrun",
```

Comment:
Why are we also waiting for mlrun-db and mlrun-ui to be ready?

Reply:
ensure application is patched without side effects